lib/docs/token-return-types.md

```diff
@@ -0,0 +1,10 @@
+# Token Return Types
```

Comment:
Can you please add some text explaining why the table is the way it is. Add what sending `openid`, `profile` would mean and what sending `client_id` as `scope` was; how both these concepts are now addressed with this new PR.

Reply:
Yes, should get that done today or tomorrow at the latest!